internal/repository/aircraft.go

```diff
@@ -54,15 +55,15 @@ func (a aircraft) Update(aircraft model.Aircraft) (model.Aircraft, error) {
 	return aircraft, nil
 }
 
-func (a aircraft) DeleteByID(id uint) error {
-	if _, err := a.GetByID(id); err != nil {
-		return err
-	}
-
-	result := a.db.Delete(&model.Aircraft{}, id)
+// Kolejny raz wykonujemy usuwanie do bazy nie wiedząc czy samolot istnieje
+func (a aircraft) DeleteByID(userID, id uint) error {
```

Comment:
Method name does not contains `userID`. It should be named `DeleteByUserIDAndID`